teams/delius-iaps/components/files/IAPSCMSIF.xml

```diff
@@ -0,0 +1,90 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
```

Comment:
Looks good but could you share where you got this file from or the knowledge base which helped you put this together?

Reply:
This was taken from from the old stage environment. These files have never been maintained by us, or the team previously in charge of the infrastructure. To our knowledge the files would be manually edited by the service team at i2n/necsws.

Reply:
To add to this, the application comes bundled with a sample config file, which contains fewer DEFPROGTEAM entries.